config/locales/administrate.fr.yml

```diff
@@ -6,6 +6,8 @@ fr:
       destroy: Supprimer
       edit: Modifier
       show: Détails
+      new: Nouveau
+      back: Arrière
     controller:
       create:
         success: "%{resource} a été correctement créé(e)."
```

Comment:
Correct here, because the feminine `(e)` form is put into parenthesis, which is fairly comon practice when the genre is unknown.